frontend_tests/node_tests/message_edit.js

```diff
@@ -79,6 +79,10 @@ run_test("get_editability", () => {
     // is true, we can edit the topic if there is one.
     message.type = "stream";
     assert.equal(get_editability(message, 45), editability_types.TOPIC_ONLY);
+    // Right now, we prevent users from editing widgets.
+    message.submessages = ["/poll"];
```

Comment:
I would also be sure to test the case where you have a "/poll" message and it's been too long to edit the topic.